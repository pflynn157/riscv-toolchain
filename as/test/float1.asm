fadd.s f1, f0, f0
fadd.s f2, f3, f4
