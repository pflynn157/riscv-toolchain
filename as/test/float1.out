000070d3
0041f153
